doc/api/deprecations.md

```diff
@@ -3056,6 +3056,24 @@ This method was deprecated because it is not compatible with
 
 Use [`buffer.subarray`][] which does the same thing instead.
 
+### DEP0159: `_getActiveRequests` and `_getActiveHandles`
```

Comment:
```suggestion
### DEP0159: `process._getActiveRequests()` and `process._getActiveHandles()`
```
Most of the other deprecation titles include the global object the deprecated functions belongs to. Also the `()` makes it clear that it's a function.

Reply:
Updated